### PR TITLE
Add new person instance segmentation model

### PR DESCRIPTION
**Pull request by DmitriySidnev**
This is an instance segmentation model trained for one class - person.

### CODE REVIEW

**Comment by vladimir-dudnik** [comment]
@DmitriySidnev what is the original for sample pictures?
cc @yssemaev

**Comment by DmitriySidnev** [comment]
@vladimir-dudnik, these images from MS COCO dataset.

**Comment by Wovchena** [comment]
Jenkins please retry a build

**Review by eaidova — APPROVED** [review]

**Review by vladimir-dudnik — APPROVED** [review]

File: data/dataset_definitions.yml [hunk 1]
```diff
@@ -155,6 +155,16 @@ datasets:
       - name: AP
         type: coco_orig_keypoints_precision
 
+  - name: ms_coco_mask_rcnn_person
```
Comment (eaidova) [inline data/dataset_definitions.yml:158]:
```suggestion
  - name: COCO2017_mask_rcnn_person
```

**Pull request merged**